src/bridge/bridgemain.h

```diff
@@ -1171,8 +1171,9 @@ typedef enum
     GUI_UPDATE_GRAPH_VIEW,          // param1=unused,               param2=unused
     GUI_SET_LOG_ENABLED,            // param1=bool isEnabled        param2=unused
     GUI_ADD_FAVOURITE_TOOL,         // param1=const char* name      param2=const char* description
-    GUI_ADD_FAVOURITE_COMMAND,      // param1=const char* command   param2=const char* shortcut
     GUI_SET_FAVOURITE_TOOL_SHORTCUT,// param1=const char* name      param2=const char* shortcut
+    GUI_ADD_FAVOURITE_COMMAND,      // param1=const char* command   param2=const char* shortcut
+    GUI_SET_FAVOURITE_COMMAND_SHORTCUT,      // param1=const char* command   param2=const char* shortcut

```

Comment:
You cannot add messages in the middle. It breaks binary compatibility (just move it to the end)

Reply:
> You cannot add messages in the middle. It breaks binary compatibility (just move it to the end)

Since I don't know QT, the function I want has not been tested successfully :(
src/gui/Src/Gui/MainWindow.cpp checksum error